src/main/java/org/folio/innreach/domain/entity/TransactionItemHold.java

```diff
@@ -16,4 +18,15 @@
 @ToString
 public class TransactionItemHold extends TransactionHold {
 
+  @Getter(AccessLevel.NONE) @Setter(AccessLevel.NONE)
+  @Column(name = "author")
+  private String authorItemHold;
```

Comment:
since this field going to be common for all the transaction types, move it to TransactionHold